config/initializers/sufia.rb

```diff
@@ -27,8 +27,8 @@
   config.resource_types = {
     "Book" => "Book",
     "Book Chapter" => "Book Chapter",
-    "Conference\/Workshop Poster" => "Conference\/Workshop Poster",
-    "Conference\/Workshop Presentation" => "Conference\/Workshop Presentation",
+    "Conference\/workshop Poster" => "Conference\/workshop Poster",
```

Comment:
We don't want to end up with "workshop Poster", do we? Not sure what's the intention here.